src/actions/utils/download-helpers.ts

```diff
@@ -1,16 +1,24 @@
 import InvoiceModel from '../../components/invoice/models/InvoiceModel';
 import {buildUrl} from './fetch';
 
+export type DownloadAttachmentModelTypes = {
+  invoice: string;
+  quotation: string;
+  client: string;
+  'project_month_overview': string;
```

Comment:
Ter info: underscore is een valid JS prop name char, de key hoeft dus niet in quotes. Dit is bijvoorbeeld wel het geval wanneer er `-` gebruikt wordt

Reply:
Yes dat weet ik maar anders moest ik eslint uitschakelen voor die lijn code en heb dat dus gewoon zo opgelost.